src/helpers/ratings.js

```diff
@@ -0,0 +1,29 @@
+import fetch from "node-fetch";
+const API_REVIEWS = "https://plankton-app-xhkom.ondigitalocean.app/api/reviews";
+const IMDB_API = "https://www.omdbapi.com/?apikey=81d6c4d0";
+
+export async function loadMovieRatings(id) {
+  const res = await fetch(`${API_REVIEWS}?filters[movie]=${id}&populate=movie`);
```

Comment:
Hard-coding `fetch` like this makes it very difficult to test a function like this. I can see that you have already figured this out and replaced it with an API adapter in the final version.